src/Reshuffle.ts

```diff
@@ -6,6 +6,24 @@ import { BaseConnector, BaseHttpConnector, EventConfiguration } from 'reshuffle-
 import { createLogger } from './Logger'
 import { Logger, LoggerOptions } from 'winston'
 import http from 'http'
+import fs from 'fs'
+
+function getReshufflePackagesList(directory = __dirname): Record<string, string> {
```

Comment:
Suggestion: When I read the function name, I thought it would only be the `reshuffle` packages :) So maybe we can name it `getInstalledPackages` or something like this?